junifer/markers/crossatlas_functional_connectivity.py

```diff
@@ -0,0 +1,164 @@
+"""Provide marker class to calculate cross-atlas FC."""
+
+# Authors: Leonard Sasse <l.sasse@fz-juelich.de>
+#          Kaustubh R. Patil <k.patil@fz-juelich.de>
+# License: AGPL
+
+from typing import Any, Dict, List, Optional
+
+import pandas as pd
+
+from ..api.decorators import register_marker
+from ..storage import BaseFeatureStorage
+from ..utils import logger
+from .base import BaseMarker
+from .parcel_aggregation import ParcelAggregation
+from .utils import _correlate_dataframes
+
+
+@register_marker
+class CrossAtlasFC(BaseMarker):
+    """Class for calculating parcel-wise correlations between two atlases.
+
+    Parameters
+    ----------
+    atlas_one : str
+        The name of the first atlas.
+    atlas_two : str
+        The name of the second atlas.
+    aggregation_method : str, optional
+        The aggregation method (default "mean").
+    correlation_method : str, optional
+        Any method that can be passed to:
+        https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.corr.html
```

Comment:
Can you use a ref here?
I think something like this should work:
`` :func:`pandas.DataFrame.corr` ``

Reply:
will try it out.